src/main/preload/lyrics.ts

```diff
@@ -0,0 +1,17 @@
+import { IpcRendererEvent, ipcRenderer } from 'electron';
+import { QueueSong } from '/@/renderer/api/types';
+
+const fetchLyrics = (song: QueueSong) => {
+  ipcRenderer.send('lyric-fetch', song);
+};
+
+const getLyrics = (
+  cb: (event: IpcRendererEvent, songName: string, source: string, lyric: string) => void,
+) => {
+  ipcRenderer.on('lyric-get', cb);
+};
```

Comment:
These two function names are a bit confusing as to what they actually do.

`fetchLyrics` -- fetches lyrics from remote sources as opposed to from the server. Rename this to `fetchRemoteLyrics` or similar

`getLyrics` -- listens for and retrieves the lyrics response from the remote sources. getLyrics is already used in the api controller, so rename to `retrieveRemoteLyrics` or similar

Reply:
Thinking about it again, the listener function should probably be be `remoteLyricsListener` since that makes it also sound like a fetch function. In the `mpv-player` preload, I had all of the listener functions under a separate named export called `mpvPlayerListener` so it was easy to identify. Since it's combined here, adding `listener` somewhere in the name should be used so it can be easily identified as such.